### ISSUE TITLE
[EN][Quest][TBC] Maintaining the Sunwell Portal (daily)

### ISSUE THREAD

**Issue by kzlatkov** [Atlantiss/BugTracker#4835]
http://www.wowhead.com/quest=11514/maintaining-the-sunwell-portal

Upon killing the mobs in the provided area they drop http://www.wowhead.com/item=34248/bashir-phasing-device which when used transfers you "Out of Phase". When in this state there should be http://www.wowhead.com/item=34246/smuggled-mana-cell spawned, also http://www.wowhead.com/npc=24917/phase-wyrm are supposed to spawn and able to kill and drop the Mana Cells.

Currently (Revision:4647) neither the Mana Cells spawn, nor the Phase Wyrms.

**Comment by Noviush** [comment]
Still not repaired... :(

**Comment by Reisha123** [comment]
Needs fix. over 6 months and nothing done.

**Comment by Malylolek** [comment]
Quest cata > other quest, It's not like they dont do anything, just have more immportant thinks for fix then this quest.

**Comment by Reisha123** [comment]
Yeah... for 2 years. They need to focus on cata content sure, but they are not doing anything besides that.

**Comment by Malylolek** [comment]
http://atlantiss.eu/changelog

**Comment by Reisha123** [comment]
cough 1 year waiting for sunwell, 2 years and a half waiting for Vash'jir.

**Comment by pelekon** [comment]
@Reisha123 if it hurts you so much here are our tutorials, gl with scripting it http://community.atlantiss.eu/index.php?/forum/139-tutorials/

**Comment by aendry** [comment]
still broken

**Comment by Thunderlmao** [comment]
As of 25/09/2017 this quest is not working.

The mana cells nor mana wrymms spawn.
